### PR TITLE
Improve nocli output logging

### PR DESCRIPTION
**Pull request by corbin-phipps**
### Type

- [ ] Bug fix
- [ ] Feature addition
- [x] Feature update
- [ ] Breaking change
- [ ] Non-functional change
- [ ] Documentation
- [ ] Infrastructure

### Goals

This PR cleans up the logging for nocli output to only display necessary/important data to the console. Verbose logs are still kept in the generated log file.

My intention was to keep FATAL, ERROR, WARNING, and INFO logging displayed to the console (so, if no errors, then just INFO logs would show), and keep all of those logs + DEBUG and VERBOSE logs in the generated log file.

### Technical Details

- Added separate PLOG loggers for the file appender and the color console appender.
- Changed std::cout and std::cerr to PLOG_*.
- Altered verbosity levels where appropriate.
- Reformatted the ToString() output for ranging data notifications.

### Test Results

Verified that all verbose logs are in the generated log file while logs only up to PLOG_INFO are displayed to the console when running nocli.

Below is example output from nocli:
image

### Reviewer Focus

I am open to suggestions for altering log severity levels. The changes I made were ones that made sense to me, but they can be changed as desired.

### Future Work

None.

### Checklist

- [x] Build target `all` compiles cleanly.
- [x] clang-format and clang-tidy deltas produced no new output.
- [x] Newly added functions include doxygen-style comment block.

### CODE REVIEW

**Comment by abeltrano** [comment]
> Below is example output from nocli: image

Beautiful 👍

File: windows/tools/nocli/Main.cxx [hunk 1]
```diff
@@ -59,15 +59,25 @@ CtrlHandler(DWORD controlType)
...
-    plog::RollingFileAppender<plog::TxtFormatter> rollingFileAppender(logging::GetLogName("nocli").c_str());
-    plog::DebugOutputAppender<plog::TxtFormatter> debugAppender;
-    plog::ColorConsoleAppender<plog::MessageOnlyFormatter> colorConsoleAppender{};
-    plog::init(plog::verbose, &rollingFileAppender)
-        .addAppender(&debugAppender)
-        .addAppender(&colorConsoleAppender);
+    static plog::ColorConsoleAppender<plog::MessageOnlyFormatter> colorConsoleAppender{};
+    plog::init<static_cast<int>(LogInstanceId::Console)>(plog::info, &colorConsoleAppender);
+
+    static plog::RollingFileAppender<plog::TxtFormatter> rollingFileAppender(logging::GetLogName("nocli").c_str());
```
Comment (abeltrano) [inline windows/tools/nocli/Main.cxx:77]: Why are these `static`?

**Review by abeltrano — APPROVED** [review]

File: windows/tools/nocli/Main.cxx (hunk 1 above)
Comment (corbin-phipps) [reply windows/tools/nocli/Main.cxx:77]: Oh I just noticed that that's what the PLOG github examples use, but they don't need to be

File: lib/uwb/protocols/fira/FiraDevice.cxx [hunk 2]
```diff
@@ -202,14 +202,14 @@ std::string
...
     std::ostringstream ss{};
-    ss << "SlotIndex: " << +SlotIndex << ", "
-       << "Distance: " << Distance << ", "
-       << "Status: " << ::ToString(Status) << ", "
-       << "Peer Mac Address: " << PeerMacAddress << ", "
-       << "Line Of Sight Indicator: " << magic_enum::enum_name(LineOfSightIndicator) << ", "
-       << "Angle of Arrival Azimuth: " << AoAAzimuth << ", "
-       << "Angle of Arrival Elevation: " << AoAElevation << ", "
-       << "Angle of Arrival Destination Azimuth: " << AoaDestinationAzimuth << ", "
+    ss << "SlotIndex: " << +SlotIndex << "\n"
```
Comment (abeltrano) [inline lib/uwb/protocols/fira/FiraDevice.cxx:205]: For future consideration, we might have a "compact" string representation that packs all of the data in a single line with acronyms, something like:

```
Id: 287454020 Peer: bb:bb SEQ: 0 LoS D: 10 AZ: -180 (100) E: -180 (100) AD: 180 (100) ED: 90 (100)
```

Then compact/normal/detailed formatting can be configured as a nocli command line option. I documented this in #304.

File: windows/tools/nocli/Main.cxx (hunk 1 above)
Comment (corbin-phipps) [reply windows/tools/nocli/Main.cxx:77]: From the github:
image

**Pull request merged**